src/RageMath.cpp

```diff
@@ -673,6 +673,7 @@ int LuaFunc_create_bezier(lua_State* L)
 {
 	RageBezier2D* bezier= new RageBezier2D;
 	bezier->PushSelf(L);
+	delete bezier;
```

Comment:
NO!  You can't delete that bezier because it's being passed to the lua code for the lua to use!  The bezier object has a destroy function so it can be deleted when the lua code is done with it.